app/uk/gov/hmrc/tradereportingextracts/controllers/ReportStatusController.scala

```diff
@@ -39,6 +41,7 @@ class ReportStatusController @Inject() (
     with Logging {
 
   def notifyReportStatus(): Action[AnyContent] = Action.async { request =>
+    implicit val hc: HeaderCarrier  = HeaderCarrierConverter.fromRequest(request)
```

Comment:
```suggestion
  def notifyReportStatus(): Action[AnyContent] = Action.async { implicit request =>
```
I think that this should work for you, as this conversion will be used:
https://github.com/hmrc/bootstrap-play/blob/be456eb0efe79349bb9163091e15c5aff482c6f4/bootstrap-backend-play-30/src/main/scala/uk/gov/hmrc/play/bootstrap/backend/controller/BackendController.scala#L38

Reply:
Thanks for that David! Handled implicitly now :)